### PR TITLE
.github folder with some basic issue and pull request templates

### PR DESCRIPTION
**Pull request by Hoi15A**
I thought it would be nice if we could have some basic templates that appear in issues and pull requests in case someone who is not part of the maintainers wants to create an issue or pull request.

I have written a basic first draft, please review.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<!-- IMPORTANT
+Before creating an issue, please check that your version of Tarchive is up to date and that there is no other open issue.
+If you are making a feature request please include a description of said feature as well as examples if necessary.
+If you are reporting a bug please include the following:
+-->
+- **Intended behaviour**
+
+- **Actual behaviour**
+
+- **How to reproduce**
```
Comment (fuerbringer) [inline .github/ISSUE_TEMPLATE.md]: I think it's not clear enough wether or not the issue submitter is supposed to write something inbetween the bulletpoints.

So is this just a checklist? Or is the issuer supposed to write something inbetween the bulletpoints? If yes, then I'd add another newline inbetween them or maybe even have an example text like this:
```
- **Intended behaviour**
<Describe the intended behaviour here.>
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+Before you submit a pull request please check that your code is functional and has as little bugs as possible. Then please list the following:
+
+- [ ] Functionality that has been added or removed
+- [ ] Issue that is being addressed (if there is one)
```
Comment (fuerbringer) [inline .github/PULL_REQUEST_TEMPLATE.md]: Maybe also add:
`- [ ] Milestone that is being worked towards (if there is one that applies)` ?

**Comment by Hoi15A** [comment]
Other than the absolute mess I have created in this PR. What do you think of the changes? @fuerbringer

**Comment by fuerbringer** [comment]
The changes related to the `.github` files are alright. However im not sure what to do about the huge mess.

**Comment by Hoi15A** [comment]
About the mess, it seems like the pull would merge correctly, so currently the options are:

1. Merge this pull
2. Close this one and create a new one
3. Close this pull and commit directly to master

**Pull request merged**